confignode/src/main/java/org/apache/iotdb/confignode/client/SyncConfigNodeClientPool.java

```diff
@@ -83,6 +95,52 @@ public TSStatus applyConfigNode(TEndPoint endPoint, TConfigNodeLocation configNo
         .setMessage("All retry failed.");
   }
 
+  public TSStatus removeConfigNode(
+      List<TConfigNodeLocation> configNodeLocations, TConfigNodeLocation configNodeLocation) {
+    // TODO: Unified retry logic
+    for (TConfigNodeLocation nodeLocation : configNodeLocations) {
```

Comment:
> Add some comment for this meaning, is this used to handle the brain-split? which means that the RemovedConfigNode may not exist in some confignode?

Because unknown Leader, try to connect to the ConfigNode in confignode_list to obtain the leader information.